install.sh

```diff
@@ -24,19 +36,27 @@ HYPRLAND_PATH="$HYPRLOAD_PATH/include/hyprland"
 progress "[3/7] Setting up hyprland source in $HYPRLAND_PATH"
 
 if [ -d "$HYPRLAND_PATH" ]; then
-    git -C "$HYPRLAND_PATH" pull
+	info "Updating existing hyprland source"
+	git -C "$HYPRLAND_PATH" pull origin main
```

Comment:
I noticed git pull doesn't always seem to work (when already checked out to a specific commit), specifying `origin main` fixed that.